modules/settings/assets/js/hooks/use-settings.js

```diff
@@ -0,0 +1,55 @@
+import { useState, createContext, useContext } from '@wordpress/element';
+
+/**
+ * Context Component.
+ */
+const SettingsContext = createContext( null );
+
+export function useSettings() {
+	return useContext( SettingsContext );
+}
+
+const SettingsProvider = ( { children } ) => {
+	// Notification
+	const [ showNotification, setShowNotification ] = useState( false );
```

Comment:
I'd say all those notification things can be moved into a new context to free up this one.
Similar to `NotificationsContext` in IO.

Reply:
Done